src/inference/ess.jl

```diff
@@ -137,24 +137,24 @@ function (ℓ::ESSLogLikelihood)(f)
     return getlogp(varinfo)
 end
 
-function DynamicPPL.tilde_assume(rng::Random.AbstractRNG, ctx::DefaultContext, sampler::Sampler{<:ESS}, right, vn, inds, vi)
+function DynamicPPL.tilde_assume(rng::Random.AbstractRNG, ctx::DefaultContext, sampler::Sampler{<:ESS}, right, vn, vi)
```

Comment:
I would assume these changes break support of DynamicPPL 0.15 so it should be removed from the compat entry?

Reply:
Ah yes, good catch :+1: